### ISSUE TITLE
Fix `image_expansion` gallery example

### ISSUE THREAD

**Issue by garrettwrong** [ComputationalCryoEM/ASPIRE-Python#697]
Was checking the docs build for a PR and found that we probably need to update this example.

I think either add `abs` to `Image` or change the associated example to use `asnumpy`. Can discuss. Both very minor, but we'll need to fix before release...

```
Unexpected failing examples:
/Users/gbwright/work/ASPIRE-Python.no_rots/gallery/tutorials/image_expansion.py failed leaving traceback:
Traceback (most recent call last):
  File "/Users/gbwright/work/ASPIRE-Python.no_rots/gallery/tutorials/image_expansion.py", line 59, in <module>
    fb_meanmax = np.mean(np.max(abs(fb_images - org_images), axis=(1, 2)))
TypeError: bad operand type for abs(): 'Image'
```

**Comment by garrettwrong** [comment]
If we add `__abs__`, then it isn't clear whether to return numpy array or `Image`.  If `Image` then we end up needing `asnumpy` or something later anyway...

So leaning towards just changing the example unless objections.

**Comment by garrettwrong** [comment]
Yeah, just going to use `asnumpy`.  Should be covered in #696

**Comment by garrettwrong** [comment]
Closed by #696
